src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-gallery-overlay.html.twig

```diff
@@ -0,0 +1,140 @@
+{#
+  Parameters:
+  - "overlay" (object)
+    - "close" (object)
+    - "previous" (object)
+    - "next" (object)
+    - "counter_separator"
+    - "download" (object)
+    - "share" (object)
+  - "item" (object)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+{% set component = 'gallery-overlay' %}
+{% set message = '' %}
+
+{% set no_close_label_case = false %}
+{% set no_close_icon_case = false %}
+{% set no_previous_label_case = false %}
+{% set no_previous_icon_case = false %}
+{% set no_next_label_case = false %}
+{% set no_next_icon_case = false %}
+{% set no_download_label_case = false %}
+{% set no_download_icon_case = false %}
+{% set no_download_path_case = false %}
```

Comment:
Missing initial value definitions with comparison to https://github.com/ec-europa/ecl-twig/pull/467/files#diff-138075bb609d50a031d5240f4d7a5011R77

Please add the missing in order to keep the good order started here making the flags visible at first look.